src/boardlib/api/aurora.py

```diff
@@ -580,5 +581,115 @@ def logbook_entries(board, username, password, grade_type="font", db_path=None):
     
     return full_logbook_df
 
+def bids_logbook_entries_agg(board, token, user_id, db_path=None):
```

Comment:
My suggestion in the other PR was that we could refactor that PR to adjust the params to get the output needed for the Climbdex changes. It looks like this PR just duplicates everything. E.g. we could refactor bids_logbook_entries to take a token instead of user/password and we can parametrize which entries are displayed in the output.

Seems like there is a lot of repeated code here that will be difficult to maintain in the future, and it's difficult to understand why it is this way and what might break if we made changes.

Let me know if I'm missing something here and there is a reason why both features can't use the same underlying functions with slightly different parameters

Reply:
makes sense, we tried our best to refactor the function, see comment below.